.pre-commit-config.yaml

```diff
@@ -7,19 +7,11 @@ repos:
     rev: v4.3.17
     hooks:
       - id: isort
-  - repo: https://github.com/ambv/black
-    # we need to use this commit or later, to pickup the fix that splits
-    # trailing comma syntax into call vs def, both of which weren't implemented
-    # until python 3.6 (3.5 has just the call syntax) and we need to support
-    # 3.5 for a few months.
-    rev: cea13f498418784e22f8fbd78db3f9240a2bad11
+  - repo: https://github.com/psf/black
+    rev: 40e8b3a231bade22d838858435a33d60a8325306
     hooks:
       - id: black
```

Comment:
@cpcloud 

I changed the `.pre-commit-config.yaml` applying the suggestion at https://github.com/psf/black/issues/438#issuecomment-522114098

I tested it in a different branch and it seems it worked.